.github/scripts/update_docs.py

```
@@ -0,0 +1,213 @@
+    for f in files:
+        print(f"    {f}")
+    print()
+ 
+    client = anthropic.Anthropic(api_key=os.environ["ANTHROPIC_API_KEY"])
+ 
+    errors = []
+    for filepath in files:
+        print(f"Processing: {filepath}")
+        try:
+            update_file(client, filepath)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)